runconfig/parse.go

```diff
@@ -187,6 +188,13 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 			MemorySwap = parsedMemorySwap
 		}
 	}
+	if *flSwappiness < 0 || *flSwappiness > 100 {
+		return nil, nil, cmd, fmt.Errorf("invalid value:%d. valid memory swappiness range is 0-100", *flSwappiness)
+	}
+	// This is to ensure that we do not try to override swappiness value for default case
+	if *flSwappiness == 60 {
+		*flSwappiness = -1
+	}
```

Comment:
By default, swappiness will be inherited from parent, so if parent is 50, and you set `--memory-swappiness=60` won't set swappiness to 60 as user may expected.